lib/game/base_game.dart

```diff
@@ -86,6 +89,11 @@ class BaseGame extends Game {
     _addLater.add(c);
   }
 
+  /// Removes a component from the components list
+  void remove(Component c) {
```

Comment:
Maybe I'm wrong and it's already possible with your PR, but we should be able to removeElement from ComposedComponent too. 
There is an add method in ComposedComponent, shouldn't we have a remove method too?

Reply:
Composed component is something that is going to be removed on 1.0.0, but you are correct, I should add the method there for now as well, thanks for pointing this.

Reply:
Is there a preview of the 1.0.0 yet? :)

Reply:
You can check there PR with the milestone 1.0.0.

You can also check the projects here on github, there is a board there with what we are planning.

But be aware that this is very much in progress yet, it may be some time yet before we are able to release.